website/docs/getting-started/setup/instance-configuration/custom-mongodb-redis.md

```diff
@@ -1,38 +1,83 @@
 ---
 sidebar_position: 3
+toc_max_heading_level: 4
 ---
-# Custom MongoDB & Redis
+# External MongoDB and Redis
 
-Appsmith runs it's own instances of MongoDB database for storing all application data, and Redis for storing session information. It's easy to switch to externally running instances of these services, by just changing the relevant environment variables.
+Appsmith runs embedded instances of MongoDB and Redis. Appsmith uses embedded MongoDB to store application data and Redis to manage sessions. You can switch to external instances by changing the configuration. This page provides steps to set up Appsmith with external MongoDB and Redis. With external MongoDB and Redis, you can manage your application's scalability and performance.
 
-:::info Important
-If you are currently using an older version of MongoDB, it's important to upgrade to version 5.0 before connecting Appsmith to an external MongoDB instance. For more information, follow the steps detailed on MongoDB official documentation to [Upgrade a Replica Set to 5.0](https://www.mongodb.com/docs/manual/release-notes/5.0-upgrade-replica-set/).
+:::caution Attention
+To use a custom MongoDB with Appsmith v1.9.0 and later, you need MongoDB version 5 or higher. Follow the steps detailed in the MongoDB official documentation to [Upgrade a Replica Set to 5.0](https://www.mongodb.com/docs/manual/release-notes/5.0-upgrade-replica-set/).
```

Comment:
```suggestion
To use a custom MongoDB with Appsmith v1.9.0 and later, you need MongoDB version 5.x . Follow the steps detailed in the MongoDB official documentation to [Upgrade a Replica Set to 5.0](https://www.mongodb.com/docs/manual/release-notes/5.0-upgrade-replica-set/).
```

Reply:
@sharat87 - Please share your thoughts here.

Reply:
No, v6 is also okay.